datamodel/extra_scripts/postgresql/105_appartements_rechten.sql

```diff
@@ -1,100 +1 @@
--- selecteer parent en child app_re's die een ondersplitsing zijn of zijn geworden
-
-CREATE OR REPLACE VIEW v_bd_app_re_app_re AS 
- SELECT b1.ref_id AS app_re_identif,
-    b2.ref_id AS parent_app_re_identif
-   FROM brondocument b1
-     JOIN brondocument b2 ON b2.identificatie = b1.identificatie
-  WHERE (b2.omschrijving = 'betrokkenBij Ondersplitsing' OR  b2.omschrijving = 'ontstaanUit HoofdSplitsing') AND b1.omschrijving = 'ontstaanUit Ondersplitsing'
-  GROUP BY b1.ref_id, b2.ref_id;
-
--- recursieve query om alle actuele appartementsrechten te vinden bij percelen
-CREATE OR REPLACE VIEW v_bd_app_re_all_kad_perceel AS 
```

Comment:
deze view wordt gebruikt in de brmo-soap service, die is hierdoor dus stuk. zie: https://github.com/B3Partners/brmo/blob/master/brmo-soap/src/main/java/nl/b3p/brmo/soap/db/EigendomInfo.java#L528-L550